package.json

```diff
@@ -4,7 +4,12 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "build": "node ./lib/scrape.js && node ./lib/generate.js"
+    "scrape": "node ./lib/scrape.js",
+    "generate": "node ./lib/generate.js",
+    "gather": "node ./lib/gather.js",
+    "clean": "mkdir -p out && rm -rf out/*",
```

Comment:
`mkdir -p out` I believe you don't need this since the directory never got deleted.